src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,75 @@
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.container {
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+}
+
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    height: 60px;
+  }
+
+  &__item {
+    margin-right: 20px;
+    position: relative;
+    height: 100%;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $c-main-text;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+
+    display: block;
+    line-height: 60px;
+
+    transition: color 0.3s;
+
+    &.is-active,
+    &:hover {
+      color: $c-accent-text;
+    }
+
+    &.is-active::after {
```

Comment:
let's add --active modifier for the links, and move these styles to those modifier
and use is-active class only in html for tests passing